qemu.go

```diff
@@ -765,6 +775,18 @@ func (q *qemu) createPod(podConfig PodConfig) error {
 
 // startPod will start the Pod's VM.
 func (q *qemu) startPod() error {
+	if q.config.Debug {
+		params := q.defaultKernelParams()
```

Comment:
wait, ```q.kernelParams``` are not logged?

Reply:
should the final kernel cmdline be logged too ?

Reply:
That actually already happens in the `govmm` package:

- https://github.com/intel/govmm/blob/master/qemu/qemu.go#L1535

It might seem a bit odd to log these params "again" (well, before the hypervisor starts), but it will allow users to know which of the kernel params that are eventually passed to govmm came from vc, the runtime and the user (config file).